rs/registry/canister/canister/registry.did

```diff
@@ -435,6 +444,7 @@ service : {
   ) -> ();
   deploy_guestos_to_some_api_boundary_nodes : (DeployGuestosToSomeApiBoundaryNodes) -> ();
   deploy_hostos_to_some_nodes : (DeployHostosToSomeNodes) -> ();
+  get_api_boundary_node_ids : () -> (GetApiBoundaryNodeIdsResponse) query;
```

Comment:
I recommend planning ahead of time (we might need to add some input arguments to this functions).

```suggestion
  get_api_boundary_node_ids : (record {}) -> (GetApiBoundaryNodeIdsResponse) query;
```

Reply:
Having this input `record {}` in candid, makes compiler unhappy about `get_api_boundary_node_ids()`, i.e. this doesn't work:
```
#[export_name = "canister_query get_api_boundary_node_ids"]
fn get_api_boundary_node_ids() {
    over(candid_one, get_api_boundary_node_ids_)
}
```